datamonster_api/lib/datamonster.py

```diff
@@ -211,35 +211,33 @@ def get_data(
         # todo: support multiple companies
         self._check_param(company=company, datasource=datasource)
 
-        params = {"companyId": company.id}
+        params = {"section_pk": company.id}
 
         if start_date is not None:
-            params["startDate"] = start_date
+            if not datasource.upperDateField:
+                raise DataMonsterError(
+                    "This data source does not yet support date queries"
+                )
+            key = "{}__gte".format(datasource.upperDateField)
+            params[key] = start_date
 
         if end_date is not None:
-            params["endDate"] = end_date
+            if not datasource.lowerDateField:
+                raise DataMonsterError(
+                    "This data source does not yet support date queries"
+                )
+            key = "{}__lt".format(datasource.lowerDateField)
```

Comment:
Are end date queries in the API exclusive? I thought they were inclusive. Let's confirm.

Reply:
I took this from [here](https://github.com/adaptivemgmt/dm-combined/blob/07dd3456525f5e32402b0208cf9c2aa8c692890b/common/helpers/oasis_helper/data_fountain_client.py#L593). Seems they are exclusive but then we subtract 1 day from the end_date, perhaps something elizabeth requested? We do this but the comment is misleading because the actual data is not changing
```
        # Change end_date to be inclusive
        df["end_date"] -= datetime.timedelta(days=1)
```

Reply:
Got it.